### DISCUSSION TITLE
Automated Folder Suggestion for PDF Saving Based on OCR Data

### DISCUSSION THREAD

**Discussion by S1S1A1** [cyanfish/naps2#403] [General]
It would be nice to have a function that automatically searches for keywords from folder names in a certain path in the OCR data when saving the PDF. These storage locations could then be displayed as a selection in the save dialog. Folder name: “Test1, Test2, Test3” > Click on Save: Search in OCR data of the document for the folder names > Suggestion of the corresponding folders.

Perhaps there is someone who can deal with this. Unfortunately, I lack the programming skills myself.

**Comment by DrMartinus** [comment]
I think that's very difficult to implement. Which word (of probably many in the scanned documents) shall be taken as basis for the suggestion? You might get a list as long as your folder tree... 
I help myself by having the folders which I need to access often and quickly by putting them in the shortcut menu of the file dialogue. At least in Linux that's easy, most desktops offer such a function for their file dialogues.
